_includes/filezilla.md

```diff
@@ -1,9 +1,31 @@
 ## General instructions
 
-The raw data of all studies published in IDR can be downloaded using using anonymous File Transfer Protocol (FTP). We recommend using the [FileZilla client](https://filezilla-project.org/download.php).
+The raw data of all studies published in IDR can be downloaded using [Globus](https://www.globus.org/) or anonymous File Transfer Protocol (FTP).
 
 You can download both whole studies as well as parts/single images.
 
+We recommend using the [Globus](https://www.globus.org/).
```

Comment:
I would not put in the link to Globus again here, because the user might click and assume he must read all the doc there - instead, you want the user to read your paragraph just below in this doc. The generic link to globus you already provided above.